docs/data_model.md

```diff
@@ -0,0 +1,591 @@
+[< Back to Navigation](../README.md)
+
+# Data model
+
+> This document will undoubtedly change and grow as we develop our understanding of the services that this application encapsulates.
+
+## 1. Introduction
+
+This document provides an overview of the data model for the NPQ application. It details the primary entities, their relationships, and the data flow within the application. This model is essential for developers, database administrators, and stakeholders to understand how data is structured, stored, and retrieved.
+
+## 2. Purpose of the Data Model
+
+The data model for storing NPQ data is designed to:
+
+- Facilitate efficient data storage and retrieval.
+- Enable the business logic of the application by structuring data relationships.
+- Support data integrity, scalability, and performance optimization.
+- Serve as a guide for developers during the implementation and maintenance phases.
+
+## 3. Entity-Relationship Diagram (ERD)
+
+This diagram represents our current understanding of the data models:
+
+```mermaid
+erDiagram
+    Application }|--|| User : ""
+    Application }|--|| LeadProvider : ""
+    Application }|--|| Course : ""
+    Application }|--|| Schedule : ""
+    Application }o--o| School : ""
+    Application }o--o| PrivateChildcareProvider : ""
+    Application }o--o| IttProvider : ""
+
+    Declaration }|--|| Application : ""
```

Comment:
Declaration also links directly to cohort and lead provider (and as a dependency on itself for superseeded by)